src/ui/pages/History/AccelerateTransactionDialog/CancelTx/CancelTx.tsx

```diff
@@ -105,8 +105,7 @@ function CancelTxContent({
         acceleratedGasPrices
       );
       const txResponse = await signTxBtnRef.current.sendTransaction({
-        transaction: tx,
-        solTransaction: undefined,
+        transaction: { evm: tx },
```

Comment:
So, cancel and speedup are available only for evm right now?
Will we even need them for solana, btw?

Reply:
Should we hide these options for solana transactions in the history completely? (Didn't find it yet, maybe remove the comment later)

Reply:
Yes, good point, we should hide them